cluster/sync.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash -ex
 
 source ./hack/common.sh
+source ${PWD}/_kubevirtci/cluster-up/cluster/ephemeral-provider-common.sh
 
-container_command=podman
-
-registry_port=$(${container_command} ps | grep -Po '(?<=0.0.0.0:)\d+(?=->5000\/tcp)' || echo "")
+container_command=$_cri_bin
```

Comment:
Why not replace `container_command` with `_cri_bin` everywhere?

Reply:
Makes sense for me too.